contrib/grinder/properties/grinder.properties.local

```diff
@@ -0,0 +1,23 @@
+grinder.script=../scripts/tests.py
```

Comment:
I thought the purpose of the ".local" file was to run the grinder, not the tests.  Shouldn't this be grinder.py?